src/Compilers/CSharp/Portable/BoundTree/BoundExpressionExtensions.cs

```diff
@@ -130,13 +132,17 @@ public static void GetExpressionSymbols(this BoundExpression node, ArrayBuilder<
                     break;
 
                 case BoundKind.BadExpression:
-                    symbols.AddRange(((BoundBadExpression)node).Symbols);
+                    foreach (var s in ((BoundBadExpression)node).Symbols)
+                    {
+                        if (s is { })
+                            symbols.Add(s);
```

Comment:
>symbols.Add(s); [](start = 28, length = 15)

nit: There is a semantic change here for error scenario. Did that affect a test?

Reply:
It was a bug not detected by any test.  I thought it was easier to fix it than try to repro it or figure out some annotation.

---
In reply to: [386572251](https://github.com/dotnet/roslyn/pull/42068#discussion_r386572251) [](ancestors = 386572251)